test/Microsoft.TemplateEngine.TestHelper/TestUtils.cs

```diff
@@ -17,10 +17,28 @@ public static string CreateTemporaryFolder(string name = "")
         }
 
         public static string GetTestTemplateLocation(string templateName)
+        {
+            return GetTestArtifactLocation(Path.Combine("test_templates", templateName));
+        }
+
+        public static string GetTestNugetLocation(string templateName = "TestNupkgInstallTemplateV2.0.0.2.nupkg")
+        {
+            string artifactsDir = GetTestArtifactLocation("nupkg_templates");
+            string nupkgPath = Path.Combine(artifactsDir, templateName);
+
+            if (!File.Exists(nupkgPath))
+            {
+                throw new FileNotFoundException($"{nupkgPath} does not exist");
+            }
+
+            return Path.GetFullPath(nupkgPath);
+        }
+
+        private static string GetTestArtifactLocation(string artifactLocation)
         {
             string codebase = typeof(TestUtils).GetTypeInfo().Assembly.Location;
             string dir = Path.GetDirectoryName(codebase);
-            string templateLocation = Path.Combine(dir, "..", "..", "..", "..", "..", "test", "Microsoft.TemplateEngine.TestTemplates", "test_templates", templateName);
+            string templateLocation = Path.Combine(dir, "..", "..", "..", "..", "..", "test", "Microsoft.TemplateEngine.TestTemplates", artifactLocation);
```

Comment:
minor: this is a bit ugly :)

Reply:
I agree. I'll think about how to improve this